pkg/forklift-api/webhooks/validating-webhook/admitters/plan-admitter.go

```diff
@@ -116,6 +116,34 @@ func (admitter *PlanAdmitter) validateWarmMigrations() error {
 	return nil
 }
 
+func (admitter *PlanAdmitter) validateLuks() error {
+	providerType := admitter.sourceProvider.Type()
+	if providerType != api.VSphere && providerType != api.Ova {
+		log.Info("Provider type (non-VSphere & non-OVA) does not support LUKS, passing")
```

Comment:
I'd expect to see a different logic here: if LUKS is specified and the provider type is not vSphere or OVA, then return an error. you probably took validateVDDK as a reference - but here, it's less obvious that LUKS are not being used when migrating from other sources